vhdbuilder/packer/windows/components-test.json

```diff
@@ -19,6 +19,11 @@
           "renovateTag": "registry=https://mcr.microsoft.com, name=windows/servercore",
           "latestVersion": "ltsc2022",
           "windowsSkuMatch": "23H2*"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=windows/servercore",
```

Comment:
You don't need to set new values in components-test.json unless you want to test a new feature of components.json